python/ray/serve/backend_worker.py

```diff
@@ -355,7 +355,8 @@ async def prepare_for_shutdown(self):
         # We set the del method to noop after succssifully calling it so the
         # destructor is called only once.
         try:
-            self.callable.__del__()
+            if hasattr(self.callable, "__del__"):
```

Comment:
Not sure if it's better to put this outside the whole try-except block, but it's probably not very consequential

Reply:
this should be totally fine!